fili-generic-example/src/test/groovy/com/yahoo/wiki/webservice/web/endpoints/TablesServletSpec.groovy

```diff
@@ -91,6 +91,7 @@ class TablesServletSpec extends Specification {
 
         List<String> metricNames = "count, added, delta, deleted, user_unique".split(',').collect { it.trim() }
         String expectedResponse = """{
+                                        "Available Intervals":[],
```

Comment:
Perhaps you should add some intervals to the physical table in the JTB so that this actually exercises the merge of intervals.